pkgs/development/tools/build-managers/bazel/bazel_3/default.nix

```diff
@@ -537,7 +546,15 @@ stdenv.mkDerivation rec {
     # The binary _must_ exist with this naming if your project contains a .bazelversion
     # file.
     cp ./bazel_src/scripts/packages/bazel.sh $out/bin/bazel
+
+    mkdir $out/share
+    cp ./bazel_src/bazel-bin/src/tools/execlog/parser_deploy.jar $out/share/parser_deploy.jar
     mv ./bazel_src/output/bazel $out/bin/bazel-${version}-${system}-${arch}
+    cat <<EOF > $out/bin/execlog
```

Comment:
I am not really happy with the name. If you install bazel in your profile, you will get a `execlog` program in your PATH... I'd really prefer `bazel-execlog` or something.

Reply:
Valid point, I've renamed it to bazel-execlog